LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED_PWM.TcPOU

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_LED_PWM" Id="{5fae4b24-b1d6-4ac5-a010-9aff4e2893db}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_LED_PWM
+(*
+    Reads an percentage of illumination determined by the user and converts it to a raw value
+    to send to the terminal output via the FB_AnalogOutput.
+    2022-5-03 Maarten Thomas-Bosum
+*)
+VAR
+	iIlluminatorINT AT %Q*: INT;
+
+    {attribute 'pytmc' := '
+        pv: PWR
+        field: ZNAM OFF
+        field: ONAM ON
+    '}
+    bLedPower AT %Q*: BOOL;
+
+    {attribute 'pytmc' := '
+        pv: CIL:PCT
+        EGU: %
+    '}
+    fIlluminatorPercent: LREAL;
+	
+    fbSetIllPercent: FB_AnalogOutput;
+END_VAR]]></Declaration>
+    <Implementation>
+      <ST><![CDATA[// Illuminator conversion to percentage
+fbSetIllPercent(
+    fReal:=fIlluminatorPercent,
+    fSafeMax:=100,
+    fSafeMin:=0,
+    iTermBits:=14, // want to set scaling to 50% duty cycle = maximum raw bits sent to terminal output
```

Comment:
This block is a pretty thin wrapper around `FB_AnalogOutput`; should it be providing control over `iTermBits`?

Reply:
I think so, because iTermBits determines the max raw value that can be sent to the output. In this case I want the max value (50% duty cycle) to be only half of what the nominal output can be.

Reply:
I think the question here is more: for generalizing this FB, shouldn't this be an input variable rather than a hard-coded 14?

Reply:
Yeah I agree, let me see if I can get it to work with the raw bits being an input variable, set at the project level.